package.json

```diff
@@ -8,11 +8,12 @@
     "url": "https://github.com/JinwookKim/cc-scraper"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "node test/chase-cash-back.js"
   },
   "author": "Jinwook Kim",
   "license": "MIT",
   "dependencies": {
+    "axios": "^0.19.2",
     "cheerio": "^1.0.0-rc.3",
     "request": "^2.88.0",
     "request-promise": "^4.2.4"
```

Comment:
Question: What is the intention behind using both axios, request, and request-promise, which seem to all be able to  do the same thing of making HTTP requests?

Reply:
ah, yeah. I need to delete request and request-promise bc they announced they'll be deprecated 2 months ago lol. RIP.